tests/cpp/multidevice.cpp

```diff
@@ -121,11 +121,8 @@ void PipelineTest::validate() {
     if (!output_tv->getDeviceMesh().has(communicator->deviceId())) {
       continue;
     }
-    auto ref_output = isSharded(output_tv) ? shardTensor(
-                                                 ref_unsharded_outputs.at(i),
-                                                 output_tv->getDeviceMesh(),
-                                                 communicator->deviceId())
-                                           : ref_unsharded_outputs.at(i);
+    auto ref_output = shardTensor(
+        ref_unsharded_outputs.at(i), output_tv, communicator->deviceId());
     auto obtained_output = outputs.at(i);
     GTEST_EXPECT_TRUE(torch::allclose(ref_output, obtained_output))
```

Comment:
I know it's pre-existing, but why `GTEST_ASSERT/EXPECT_*` instead of simply `ASSERT/EXPECT_*`?